03_launch_mgmt_cluster.sh

```diff
@@ -39,14 +36,31 @@ source lib/ironic_basic_auth.sh
 # BMO and Ironic deployment functions
 # ------------------------------------
 
+function deployIronicAndOrBMO() {
+  export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
+
+  IRONIC_DATA_DIR="${IRONIC_DATA_DIR:-/opt/metal3/ironic/}"
+  IRONIC_AUTH_DIR="${IRONIC_AUTH_DIR:-${IRONIC_DATA_DIR}auth/}"
+
+  sudo mkdir -p "${IRONIC_DATA_DIR}"
+  sudo mkdir -p "${IRONIC_AUTH_DIR}"
+
+  sudo chown -R "${USER}:$(id -gn)" "${IRONIC_DATA_DIR}"
+  sudo chown "${USER}:$(id -gn)" "${IRONIC_AUTH_DIR}"
+
+  sudo chmod -R 755 "${IRONIC_DATA_DIR}"
+  sudo chmod 755 "${IRONIC_AUTH_DIR}"
+
+  go build -o "${BMOPATH}/tools/deploy" "${BMOPATH}/tools/deploy.go"
+  "${BMOPATH}/tools/deploy" "${1}" "${@:2}"
```

Comment:
Whats the point of this.. I don't see any manipulation of the vars, so `"$@"` would do the same ?

Reply:
Yes `"$@"` would do the same. I changed it.